service/src/main/java/com/codeforcommunity/processor/SiteProcessorImpl.java

```diff
@@ -93,6 +108,261 @@ public void recordStewardship(
   }
 
   @Override
+  public void updateSite(JWTData userData, int siteId, UpdateSiteRequest updateSiteRequest) {
+    checkSiteExists(siteId);
+
+    SiteEntriesRecord lastEntry =
+        db.selectFrom(SITE_ENTRIES)
+            .where(SITE_ENTRIES.SITE_ID.eq(siteId))
+            .orderBy(SITE_ENTRIES.UPDATED_AT.desc())
+            .limit(1)
+            .fetchOne();
+
+    SiteEntriesRecord record = db.newRecord(SITE_ENTRIES);
```

Comment:
If it helps, I think you can also do `db.newRecord(SITE_ENTRIES, lastEntry)` to create a copy. Double check that it does what you want if you're going to use it though.

Reply:
+1